Add a (structured) description our community

itrich opened the pull request:
This PR adds a structured description of our community. What types of organizational units do we host and what are the different teams, SIG and WG with their respective mission statements.

Huge thanks to @alexander-diab for a first shot on the SIG Market mission statement.

This is WIP. I invite all team or SIG owners to add a mission statement for the groups they're leading.
@fkr @garloff @reqa @maxwolfs 

We will use this structured data to render a description on the various pages we own. Thank you for contributing to make our community and project more clear to the public. Feedback is highly appreciated!

fkr commented:
Thanks, that is really good. Will work to add items to it today.
What I like to throw out once: Do we _really_ need the differentiation between SIG and WG? I don't comprehend why we need it.

itrich commented:
> Thanks, that is really good. Will work to add items to it today. What I like to throw out once: Do we _really_ need the differentiation between SIG and WG? I don't comprehend why we need it.

I'm happy to start the discussion whether or not we need a group of type _Working group_. My initial motivation was to distinguish between one-shot groups with a specific task and persisting groups. But I agree that this adds unneeded complexity.

itrich commented:
> From my perspective this creates an artifical "high burden" to create a SIG, while its only difference to a working group is the name. The Working Group Open Operations could've been just a SIG, since it is 'Special Interest' and features the same other properties (someone who wears the coordinators hat, a meeting slot, and so on).

I agree with you. Even the description reads the very same. I removed the type `WG` from the list and moved the OpenOps to the SIG list.

garloff commented:
I assume you'd like me to push to this branch, adding the PB, SIG-Std, and Team-Container?

fkr commented:
@garloff correct.

fkr commented:
@itrich I'll actually take this topic into the next team meetings, since I want to have a description that is written by the team and not just by myself. Basically,  I want to use 15 minutes per each of the teams where we come up with a small mission statement for the team.

itrich commented:
> @itrich I'll actually take this topic into the next team meetings, since I want to have a description that is written by the team and not just by myself. Basically, I want to use 15 minutes per each of the teams where we come up with a small mission statement for the team.

Awesome! That was the initial idea we had once in a SIG Community meetup. The most valuable outcome is not the description itself, but the better understanding the team gets by self describing their role in our open source project. Thank you for picking this up!
